### DISCUSSION TITLE
QOL Feature | Seperate Hitsounds and Global effects audio

### DISCUSSION THREAD

**Discussion by Striikerr** [ppy/osu#14529] [Ideas]
Currently in Lazer, when adjusting the audio settings. Effect changes both; the volume of the global effects (clicking on songs, hovering over buttons etc) along with the hitsounds while actually playing a beatmap. I was wondering if it'd be possible (or I guess already planned?) to separate the hitsounds into their own audio category, as I prefer to play at 100% effect (Just for the hitsounds) volume, but when navigating through song select it can be kinda deafening/ annoying after a while of pressing up/down to navigate through songs because of the audio that plays when you select one. 
To get similar results I usually play at 100% effects, 60% master, 50% music.

It'd essentially end up looking like this if implemented (and wouldn't be seen when using scroll wheel) unless manually changed it'd probably just follow the volume of the global effects.
![image](https://user-images.githubusercontent.com/33607709/131139539-826db128-5c73-40c9-bda3-a1a73c89b4df.png)

**Comment by peppy** [comment]
We'd hope to make the effect sounds not annoying enough to need to turn it off. Is it the "scratch" sound you find deafening? I believe that was intended to be a placeholder but @nekodex hasn't found a better replacement yet.

**Reply by Striikerr** [reply]
Possibly? The sound itself isn't annoying, but the repetition of it is, as someone who navigates songs using the arrow keys. I’m not sure if this would be possible, but another alternative could be to mute / duck (not sure if thats the right use of that) the sound, if you navigate in rapid succession. (I wanna say stable does this already, but i’m not at my pc to confirm)

**Reply by integerrr** [reply]
the other way around for me actually, sometimes i want to mute hitsounds without also muting the effect sounds, the effect sounds are fine to me

**Comment by AlexHgC** [comment]
Would be very useful for players who don't want hitsound but want UI sounds
